src/api/index.ts

```diff
@@ -205,3 +205,18 @@ export async function getStake(
   };
   return await client.view(payload);
 }
+
+export async function getValidatorCommission(
+  nodeUrl: string,
+): Promise<Types.MoveValue> {
+  const client = new AptosClient(nodeUrl);
```

Comment:
i add aptos client in the global state in a previous pr but hooks can't be used in regular functions, it has to be either custom hooks or function components. any suggestions on how to abstract out aptos client here? @0xmaayan